app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,356 @@
+  border-bottom: none;
+}
+
+.categories-container {
+  display: flex;
+  flex-direction: column;
+  background-color: #f0f0f0;
+  width: 100%;
+  height: 100vh;
+  overflow: hidden;
+}
+
+.categories {
+  display: flex;
+  flex-direction: column;
+  height: 75%;
+}
+
+.categories-header {
+  display: flex;
+  align-items: center;
+  background-color: #3778c2;
+  height: 12%;
+}
+
+.header {
+  align-items: center;
+  justify-content: center;
+  margin-left: auto;
+  color: #fff;
+  font-size: 1.3rem;
+}
+
+.payments-container {
+  display: flex;
+  flex-direction: column;
+  background-color: #f0f0f0;
+  width: 100%;
+  height: 100vh;
+}
+
+.payments-header {
+  display: flex;
+  align-items: center;
+  background-color: #3778c2;
+  height: 12vh;
+}
+
+.payment-header-container {
+  flex-grow: 1;
+  display: flex;
+  justify-content: center;
+}
+
+.payment-header {
+  color: #fff;
+  font-size: 1.3rem;
+}
+
+.back-button {
+  font-size: 1.3rem;
+  color: #fff;
+}
+
+.logout {
+  margin-left: auto;
+  margin-right: 10px;
+  text-decoration: none;
+  color: #fff;
+}
+
+.categories-lists,
+.payments-lists {
+  font-size: 1.1rem;
+  padding-inline-start: 0;
+  max-height: 100%;
+  width: 100%;
+  overflow-x: hidden;
+  overflow-y: scroll;
+}
+
+.categories-list {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  padding: 20px;
+  margin: 10px;
+}
+
+.icon-large {
+  font-size: 3rem;
+}
+
+.category,
+.payment {
+  display: flex;
+  gap: 40px;
+}
+
+.category-name,
+.payment-name {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+}
+
+.new-category,
+.new-payment {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: #5fb523;
+  padding: 15px;
+  margin-top: auto;
+  height: 13%;
+}
+
+.category-link,
+.payment-link {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-decoration: none;
+  color: #fff;
+  font-size: 1.2rem;
+  border: 2px solid #fff;
+  width: 100%;
+  height: 100%;
+}
+
+.uppercase-text {
+  text-transform: uppercase;
+}
+
+.payments-details {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.payments {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 10px;
+  margin-bottom: 10px;
+}
+
+.payments span {
+  font-size: 1.3rem;
+}
+
+.payments-list {
+  display: flex;
+  justify-content: space-between;
+  gap: 20px;
+  padding: 20px;
+  margin: 10px;
+  width: 100%;
+  background-color: #fff;
+}
+
+.payments-new,
+.categories-new {
+  padding: 20px;
+}
+
+.form,
+.form-name {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  font-size: 1.2rem;
+  color: #000;
+  border: none;
+}
+
+.form-group {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+}
+
+.form-control,
+.category-select {
+  padding: 10px;
+}
+
+.form-btns {
+  display: flex;
+  border: none;
+  background-color: #5fb523;
+  color: #fff;
+  font-size: 1.2rem;
```

Comment:
Your app is working well, and with awesome UI 👌 However, it will be a good idea if we can add a few tweaks to the design aspect so as to improve the look and feel. Please take note that it's one of the project's objectives that our design closely mimics the given [template design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=). Let's improve upon the design aspect by introducing the following tweaks:

- [ ] Let's ensure the `Create Category`, `Create Payment`, and all similar buttons stick to the provided design typography which includes both the color (green with white rectangular inner border) and layout (is always sticky to the bottom of the screen) like can be seen in the template design. Doing so will ensure uniformity within the design 😉 

**Your Create Category screen:**        |  **Your Create Payment:**        |  **Expected button design & position**     
:-------------------------:|:-------------------------:|:-------------------------:
<img width="800" alt="Home Page" src="https://github.com/Lawmsangi/budget-app/assets/12745474/ddabb7be-80e1-4de3-b7cf-32fdfafcb791"> | <img width="800" alt="Home Page" src="https://github.com/Lawmsangi/budget-app/assets/12745474/29575b24-4e07-42be-81cd-0e5abbb4468d"> | <img width="800" alt="Home Page" src="https://github.com/Lawmsangi/budget-app/assets/12745474/b40d9f78-1423-40c7-ae9a-698b4bd15933">